### ISSUE TITLE
typo in base package order_command() function

### ISSUE THREAD

**Issue by jar-dev** [Thinstation/thinstation#191]
Missing a '$' ;)

Only relevant if reverse order is activated.

```
--- a/ts/build/packages/base/etc/thinstation.packages
+++ b/ts/build/packages/base/etc/thinstation.packages
@@ -78,7 +78,7 @@ order_command()
                PASSWORD="'$PASSWORD'"
        fi
        if is_enabled $REVERSE_ORDER ; then
-               ORDERED_COMMAND="${SERVER_ARG}$SERVER $OPTIONS ${USER_ARG}$REMUSER {PASSWORD_ARG}$PASSWORD $MISC_OPTIONS"
+               ORDERED_COMMAND="${SERVER_ARG}$SERVER $OPTIONS ${USER_ARG}$REMUSER ${PASSWORD_ARG}$PASSWORD $MISC_OPTIONS"
        else
                ORDERED_COMMAND="$OPTIONS ${USER_ARG}$REMUSER ${PASSWORD_ARG}$PASSWORD $MISC_OPTIONS ${SERVER_ARG}$SERVER"
        fi
```

**Comment by Doncuppjr** [comment]
blockquote, div.yahoo_quoted { margin-left: 0 !important; border-left:1px #715FFA solid !important; padding-left:1ex !important; background-color:white !important; }  Nice find


Sent from Yahoo Mail for iPhone


On Thursday, October 27, 2016, 10:12 AM, jar-dev <notifications@github.com> wrote:


Missing a '$' ;)

Only relevant if reverse order is activated.
--- a/ts/build/packages/base/etc/thinstation.packages
+++ b/ts/build/packages/base/etc/thinstation.packages
@@ -78,7 +78,7 @@ order_command()
                PASSWORD="'$PASSWORD'"
        fi
        if is_enabled $REVERSE_ORDER ; then
-               ORDERED_COMMAND="${SERVER_ARG}$SERVER $OPTIONS ${USER_ARG}$REMUSER {PASSWORD_ARG}$PASSWORD $MISC_OPTIONS"
+               ORDERED_COMMAND="${SERVER_ARG}$SERVER $OPTIONS ${USER_ARG}$REMUSER ${PASSWORD_ARG}$PASSWORD $MISC_OPTIONS"
        else
                ORDERED_COMMAND="$OPTIONS ${USER_ARG}$REMUSER ${PASSWORD_ARG}$PASSWORD $MISC_OPTIONS ${SERVER_ARG}$SERVER"
        fi

**Comment by MarcusEyre** [comment]
This is fixed in pull request 203 (merged into 5.5-Stable
